.github/workflows/msbuild.yml

```diff
@@ -6,16 +6,14 @@ on:
   pull_request:
     branches: [ master ]
 
-env:
-  dotnet-version: 5.0.x # SDK Version for building Dafny; x will use the latest version of the 5.0 channel
 jobs:
   singletons:
     runs-on: ubuntu-18.04
     steps:
-    - name: Setup dotnet 5.0
-      uses: actions/setup-dotnet@v1.7.2
```

Comment:
It would be better to use a fixed version and set up dependabot to keep this updated, in the name of repeatable builds. Can we leave it for now, or at least just use the newest explicit version?